libr/bin/dbginfo.c

```diff
@@ -1,9 +1,10 @@
-/* radare - LGPL - Copyright 2009-2016 - nibble, pancake */
+/* radare - LGPL - Copyright 2009-2020 - nibble, pancake */
 
 #include <r_types.h>
 #include <r_bin.h>
 
-R_API int r_bin_addr2line(RBin *bin, ut64 addr, char *file, int len, int *line) {
+R_API bool r_bin_addr2line(RBin *bin, ut64 addr, char *file, int len, int *line) {
```

Comment:
if we change this we should also change the `RBinDbgInfo.get_line` signature to return bool.

Reply:
done

Reply:
already pushed?